lib/agris/api/messages/query_base.rb

```diff
@@ -0,0 +1,37 @@
+# frozen_string_literal: true
+module Agris
+  module Api
+    module Messages
+      class QueryBase < MessageBase
+        protected
+
+        def input_base_hash
+          {
+            :@requester => 'Otis (Agris Client)',
```

Comment:
This would just be `Agris Client` now wouldn't it, unless we want to specify an agent as we did with Ottokit?